MapboxCoreNavigation/RouteStepFormatter.swift

```diff
@@ -23,6 +23,17 @@ public class RouteStepFormatter: Formatter {
         
         let modifyValueByKey = { (key: OSRMTextInstructions.TokenType, value: String) -> String in
             switch key {
+            case .code:
+                let refComponents = value.addingXMLEscapes.components(separatedBy: .whitespaces)
+                let refsContainingShieldValue = refComponents.filter {
```

Comment:
We should only be looking up the first word in `ShieldImageNamesByPrefix`. For example, hypothetically, if the code is `City 123 I`, we shouldn’t spell out C-I-T-Y.